requirements.txt

```diff
@@ -1,2 +1,15 @@
-SQLAlchemy
-PyYAML --global-option="--with-libyaml"
+CacheControl==0.12.6
+certifi==2020.4.5.2
+chardet==3.0.4
+idna==2.9
+msgpack==1.0.0
+pkg-resources==0.0.0
```

Comment:
this is a non valid thing. Have you generated this requirements directly from what you had installed? Recommend to remove this line (have tested this myself)

Reply:
Which line was invalid?

The requirements.txt was generated directly from pip freeze (python 3.8.2 / pip 20.1.1) I and then added the --with-libyaml global option back for PyYAML.

Reply:
in github the comment is always directly below the line that is mentioned (confusing, I agree), so in this case line 6. pkg-resources is something from the OS I believe and shouldnt be in requirements. also I think the version pinning like this is a little to restrictive, minor updates should not break the script.

Reply:
This makes sense, thank you. i have updated requirements.txt and removed the version pinning and extra packages. I also took out the --global-option="--with-libyaml" aas well, because it prevents pip from installing wheels.